src/PIL/PSDraw.py

```diff
@@ -71,19 +71,18 @@ def setfont(self, font, size):
         """
         if font not in self.isofont:
             # reencode font
-            self._fp_write("/PSDraw-{} ISOLatin1Encoding /{} E\n".format(font, font))
+            self._fp_write(f"/PSDraw-{font} ISOLatin1Encoding /{font} E\n")
             self.isofont[font] = 1
         # rough
-        self._fp_write("/F0 %d /PSDraw-%s F\n" % (size, font))
+        self._fp_write(f"/F0 {size} /PSDraw-{font} F\n")
 
     def line(self, xy0, xy1):
         """
         Draws a line between the two points. Coordinates are given in
         PostScript point coordinates (72 points per inch, (0, 0) is the lower
         left corner of the page).
         """
-        xy = xy0 + xy1
-        self._fp_write("%d %d %d %d Vl\n" % xy)
+        self._fp_write(f"{xy0[0]} {xy0[1]} {xy1[1]} {xy1[1]} Vl\n")
```

Comment:
I'm not sure the new format is actually more readable here.

I would do the following instead:

```suggestion
        self._fp_write("%d %d %d %d Vl\n" % (*xy0, *xy1))
```